pki-setup.sh

```diff
@@ -187,6 +187,15 @@ if [[ ! -f $1 ]]; then
         exit
 fi
 
+# Variables and parameters
+echo ">> Setup _VAULT_DIR and fix permissions"
+_VAULT_DIR=${_VAULT_DIR:-/vault}
+chown -R vault:vault ${_VAULT_DIR}
```

Comment:
Let's put this inside a `if` check for some `SNAP*` variable, as on Ubuntu Core we won't have a `vault` user or group. Something like
```bash
if [ -z "$SNAP" ]; then
    chown -R vault:vault ${_VAULT_DIR}
    chown -R vault:vault ${_VAULT_DIR}/pki
fi
```

Reply:
Also note that I meant this check to be for detecting any kind of snap install vs docker, and not to distinguish between classic and core snap environments. But I'm fine if you want to check for classic vs ubuntu core systems too.